src/pages/about-us.js

```diff
@@ -1,15 +1,27 @@
 import Head from 'components/Head'
 import aboutUsData from '../../data/about-us'
+import { AboutUsHero, OurStory } from 'blocks'
+import get from 'utils/get'
 import PageContent from 'components/PageContent'
-import { AboutUsHero } from 'blocks'
 
 const AboutUs = () => {
-  const { meta, hero } = aboutUsData
+  const { meta, hero, ourStory } = aboutUsData
   return (
     <>
       <Head meta={meta} />
       <PageContent>
         <AboutUsHero hero={hero} />
+        <div
+          style={{
+            backgroundColor: get('colors.valhalla.700'),
+            height: 200,
+            color: 'white',
+            margin: '0 -32px',
+          }}
+        >
+          Other block to come
```

Comment:
Is this wanted?

Reply:
Yes ! That's because the 2 blocks that are already done have the same background-color, but as they have padding top + bottom it looked broken because there was a looot of space between the 2 contents. I added this in order to previsualize what it could look like with a block in between (to be developed) with an other background color :)